conf/systemd.service

```
@@ -8,7 +8,7 @@ Group=__APP__
 UMask=0002
 
 Type=simple
-ExecStart=/usr/bin/mono --debug /usr/lib/sonarr/bin/Sonarr.exe -nobrowser -data=/var/lib/__APP__
+ExecStart=/usr/bin/mono --debug /usr/lib/sonarr/bin/Sonarr.exe -nobrowser -data=__INSTALL_DIR__
```

```suggestion
ExecStart=/usr/bin/mono --debug /usr/lib/sonarr/bin/Sonarr.exe -nobrowser -data=/var/lib/__APP__
```

Feels like there's something confusing about where this app does store its data x_X Since it's related to a debian package there might be some ~good reason the data dir is supposed to be /var/lib/$app

---

using `__DATA_DIR__` seems appropriate isn't it?